@here/olp-sdk-dataservice-read/lib/CatalogsRequest.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * A class that prepare information for calls to get Schema from ConfigAPI
+ */
+export class CatalogsRequest {
+    private schema: string | undefined;
+
+    constructor() {}
+
+    public getSchema(): string | undefined {
+        return this.schema;
+    }
+
+    public withSchema(schema: string) {
```

Comment:
Please add a comment about schema and how user can use it

Reply:
done